pa11y_runner.py

```diff
@@ -80,14 +71,22 @@ def run_pa11y(file: str, output_dir: str, standard: str, runner: str) -> Union[s
     err_file.close()
```

Comment:
Use a context manager on the `report_file` and `err_file`

Reply:
Done